Tests/FeaturevisorSDKTests/InstanceTests.swift

```diff
@@ -5,6 +5,126 @@ import XCTest
 
 class FeaturevisorInstanceTests: XCTestCase {
     
+    func testEncodingEvaluationWithNilValuesReturnsValidDatafileContent() throws {
+
+        //GIVEN
+        let evaluation = Evaluation(
+            featureKey: "feature123",
+            reason: .allocated
+        )
+
+        let expectedDictionary: [String: Any] = [
+            "featureKey": "feature123",
+            "reason": "allocated",
+            "bucketValue": NSNull(),
```

Comment:
Why not "nil"?

Reply:
'nil' is not compatible with expected dictionary value type 'Any'